documentation/test-api/selecting-page-elements/selectors/using-selectors.html

```diff
@@ -851,16 +922,16 @@ <h2><a class="anchor" name="check-if-an-element-exists"></a>Check if an Element
+<div class="highlight"><pre><code class="language-js" data-lang="js"><span class="k">import</span> <span class="p">{</span> <span class="nx">Selector</span> <span class="p">}</span> <span class="k">from</span> <span class="s1">'testcafe'</span><span class="p">;</span>
 
-<span class="nx">fixture</span> <span class="err">`</span><span class="nx">Example</span> <span class="nx">page</span><span class="err">`</span>
-    <span class="p">.</span><span class="nx">page</span> <span class="err">`</span><span class="nx">http</span><span class="err">:</span><span class="c1">//devexpress.github.io/testcafe/example/`;</span>
+<span class="nx">fixture</span> <span class="s2">`Example page`</span>
+    <span class="p">.</span><span class="nx">page</span> <span class="s2">`http://devexpress.github.io/testcafe/example/`</span><span class="p">;</span>
 
-<span class="nx">test</span><span class="p">(</span><span class="s1">'My test'</span><span class="p">,</span> <span class="nx">async</span> <span class="nx">t</span> <span class="o">=&gt;</span> <span class="p">{</span>
-    <span class="kr">const</span> <span class="nx">osCount</span>            <span class="o">=</span> <span class="nx">Selector</span><span class="p">(</span><span class="s1">'.column.col-2 label'</span><span class="p">).</span><span class="nx">count</span><span class="p">;</span>
-    <span class="kr">const</span> <span class="nx">submitButtonExists</span> <span class="o">=</span> <span class="nx">Selector</span><span class="p">(</span><span class="s1">'#submit-button'</span><span class="p">).</span><span class="nx">exists</span><span class="p">;</span>
+<span class="nx">test</span><span class="p">(</span><span class="s1">'My test'</span><span class="p">,</span> <span class="k">async</span> <span class="nx">t</span> <span class="o">=&gt;</span> <span class="p">{</span>
+    <span class="kd">const</span> <span class="nx">osCount</span>            <span class="o">=</span> <span class="nx">Selector</span><span class="p">(</span><span class="s1">'.column.col-2 label'</span><span class="p">).</span><span class="nx">count</span><span class="p">;</span>
+    <span class="kd">const</span> <span class="nx">submitButtonExists</span> <span class="o">=</span> <span class="nx">Selector</span><span class="p">(</span><span class="s1">'#submit-button'</span><span class="p">).</span><span class="nx">exists</span><span class="p">;</span>
 
-    <span class="nx">await</span> <span class="nx">t</span>
+    <span class="kr">await</span> <span class="nx">t</span>
         <span class="p">.</span><span class="nx">expect</span><span class="p">(</span><span class="nx">osCount</span><span class="p">).</span><span class="nx">eql</span><span class="p">(</span><span class="mi">3</span><span class="p">)</span>
         <span class="p">.</span><span class="nx">expect</span><span class="p">(</span><span class="nx">submitButtonExists</span><span class="p">).</span><span class="nx">ok</span><span class="p">();</span>
```

Comment:
@MargaritaLoseva @miherlosev Isn't here `Attempted to run assertions on a Promise object. Did you forget to await it? If not, pass "{ allowUnawaitedPromise: true }" to the assertion options.` error?